models/event_params.go

```diff
@@ -0,0 +1,27 @@
+// Credit for The NATS.IO Authors
```

Comment:
create this struct inside the analytics package since it is not going to be in use